src/platform/forms/tests/save-in-progress/SaveInProgressIntro.unit.spec.jsx

```diff
@@ -1,7 +1,7 @@
 import React from 'react';
 // import moment from 'moment';
 import { expect } from 'chai';
-import { shallow } from 'enzyme';
+import { mount, shallow } from 'enzyme';
```

Comment:
We should look at refactoring this test if possible to not use enzyme and use RTL instead since that is now considered the best practice.

I realize that this unit test is also quite long, so I'll write a tech debt ticket to put in refinement around this.